planner/planner_test.go

```diff
@@ -0,0 +1,81 @@
+package planner_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/plan/plantest"
+	"github.com/influxdata/flux/planner"
+)
+
+// from() |> range() -> FromRange
+func TestPlanFromRange(t *testing.T) {
+	// Instantiate planner
+	physicalPlanner := planner.NewLogicalToPhysicalPlanner([]planner.Rule{
```

Comment:
It would be great to be able to start from Flux text and convert it to a logical plan.  That would make these tests much easier to read.